pkg/storage/internalstorage/register.go

```diff
@@ -66,6 +68,17 @@ func NewStorageFactory(configPath string) (storage.StorageFactory, error) {
 	if err != nil {
 		return nil, err
 	}
+	sqlDB, err := db.DB()
+	if err != nil {
+		return nil, err
+	}
+	connPool, err := cfg.getConnPoolConfig()
+	if err != nil {
+		return nil, err
+	}
+	sqlDB.SetMaxIdleConns(connPool.MaxIdleCount)
+	sqlDB.SetMaxOpenConns(connPool.MaxOpenConns)
+	sqlDB.SetConnMaxLifetime(connPool.MaxLifetime)
```

Comment:
Is consistent with the `SetConnMaxLifetime` function, named ConnMaxLifetime?

Reply:
OK, I also think it's easier to understand by this.